io.ossia.score.yml

```diff
@@ -0,0 +1,333 @@
+      - -DBUILD_SHARED_LIBS=OFF
+    sources:
+      - type: archive
+        url: https://github.com/google/re2/archive/refs/tags/2023-06-01.tar.gz
+        sha256: 8b4a8175da7205df2ad02e405a950a02eaa3e3e0840947cd598e92dca453199b
+
+  # Needed for CoAP protocol support
+  - name: libcoap
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DBUILD_SHARED_LIBS=OFF
+      - -DENABLE_WS=1
+      - -DENABLE_DTLS=0
+      - -DENABLE_EXAMPLES=0
+      - -DENABLE_DOCS=0
+      - -DENABLE_OSCORE=0
+      - -DENABLE_SERVER_MODE=0
+      - -DMAX_LOGGING_LEVEL=0
+    sources:
+      - type: archive
+        url: https://github.com/obgm/libcoap/archive/refs/tags/v4.3.4a.tar.gz
+        sha256: 45f1aabbb5f710e841c91d65fc3f37c906d42e8fc44dd04979e767d3960a77cf
+
+  # ossia uses boost
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - mkdir -p "${FLATPAK_DEST}/include/"
+      - cp -rf boost "${FLATPAK_DEST}/include/"
+    sources:
+      - type: archive
+        url: https://archives.boost.io/release/1.85.0/source/boost_1_85_0.tar.bz2
+        sha256: 7009fe1faa1697476bdc7027703a2badb84e849b7b0baad5086b087b971f8617
+
+  # HAP video decoding
+  - name: snappy
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DBUILD_SHARED_LIBS=OFF
+      - -DSNAPPY_BUILD_TESTS=0
+      - -DSNAPPY_BUILD_BENCHMARKS=0
+    sources:
+      - type: archive
+        url: https://github.com/google/snappy/archive/refs/tags/1.2.1.tar.gz
+        sha256: 736aeb64d86566d2236ddffa2865ee5d7a82d26c9016b36218fcc27ea4f09f86
+
+  # ossia uses mDNS for multiple protocol discovery mechanisms
+  - name: avahi
+    builddir: true
+    config-opts:
+      - --with-distro=none
+      - --disable-qt3
+      - --disable-qt4
+      - --disable-qt5
+      - --disable-gtk
+      - --disable-gtk3
+      - --disable-glib
+      - --disable-gobject
+      - --disable-python
+      - --disable-mono
+      - --disable-monodoc
+      - --disable-libevent
+      - --disable-libdaemon
+      - --disable-dbm
+      - --disable-gdbm
+      - --disable-manpage
+
+    post-install:
+      - make -C avahi-common install
+      - make -C avahi-client install
+
+    sources:
+      - type: archive
+        url: https://github.com/avahi/avahi/releases/download/v0.8/avahi-0.8.tar.gz
+        sha256: 060309d7a333d38d951bc27598c677af1796934dbd98e1024e7ad8de798fedda
+
+  - name: ossia-score
+    buildsystem: cmake-ninja
+    builddir: true
+    no-make-install: true
+    sources:
+      - type: archive
+        url: https://github.com/ossia/score/releases/download/v3.2.3/ossia.score-3.2.3-src.tar.xz
+        sha256: 0
+    config-opts:
+      # Note: don't use RelWithDebInfo as ossia release builds explicitly expect Release build type.
```

Comment:
so no debug symbols?

Reply:
as of now no though I can try to fix that upstream for a further version